core/templates/span.h

```diff
@@ -62,4 +62,54 @@ class Span {
 
 	_FORCE_INLINE_ constexpr const T *begin() const { return _ptr; }
 	_FORCE_INLINE_ constexpr const T *end() const { return _ptr + _len; }
+
+	// Algorithms.
+	constexpr int64_t find(const T &p_val, int64_t p_from = 0) const;
+	constexpr int64_t rfind(const T &p_val, int64_t p_from = 0) const;
+	constexpr uint64_t count(const T &p_val) const;
 };
+
+template <typename T>
+int64_t Span<T>::find(const T &p_val, int64_t p_from) const {
```

Comment:
Oop, the implementations need to be marked `constexpr` too

Reply:
Oh yeah, that explains the CI problems. I relied on AI and Clion to tell me it wasn't needed and left it out.